core/testdata/format/suspendInlineFunction.kt

```diff
@@ -0,0 +1,6 @@
+/**
+ * returns 1
+ */
+suspend inline fun foo(): Int {
```

Comment:
Maybe it worth adding a test for different than canonical order?

Reply:
done